internal/reconciler/logpipeline/otel/reconciler_test.go

```diff
@@ -544,9 +568,13 @@ func TestReconcile(t *testing.T) {
 		pipelineLock := &mocks.PipelineLock{}
 		pipelineLock.On("TryAcquireLock", mock.Anything, mock.Anything).Return(nil)
 		pipelineLock.On("ReleaseLock", mock.Anything).Return(nil)
+		pipelineLock.On("IsLockHolder", mock.Anything, mock.Anything).Return(nil)
 
 		pipelineValidatorWithStubs := &Validator{
-			PipelineLock: pipelineLock,
+			PipelineLock:       pipelineLock,
+			EndpointValidator:  stubs.NewEndpointValidator(nil),
+			TLSCertValidator:   stubs.NewTLSCertValidator(nil),
+			SecretRefValidator: stubs.NewSecretRefValidator(nil),
```

Comment:
Can you please review the todos and add missing so that we can delete all TODOs: https://github.com/kyma-project/telemetry-manager/pull/2140/files#diff-998f8849e285175fc31701cd4c920d405dc5d29c5cd839084ec6a0bc20e54cc7R542-R547

Reply:
TODOs removed, only self-monitor left